docs/development/hot-reloading-web.md

```diff
@@ -29,7 +30,8 @@ More exact instructions can be found in the
 
 The basic gist is:
 
-1. Make sure the `web` service is commented out in `docker-compose.yml`
+1. Make sure the `web` service is set to use 0 replicas in the
+   `docker-compose.override.yml`/`docker-compose.build.yml` file
```

Comment:
I found this section a bit unclear- i would preffer something like this-
```suggestion
1. Make sure only 1 instance of the `web` service is running. Configure the docker variety to run in the `docker-compose.override.yml` & `docker-compose.build.yml` files, or run it outside of docker.
```

Reply:
This section is only for running it outside of docker, so I'm more leaniant to having it as-is. I'll try to make it clearer though

Reply:
I've updated it. Take a look now